lib/blueprinter/view_collection.rb

```diff
@@ -29,8 +29,10 @@ def fields_for(view_name)
       (identifier_fields + sorted_fields).reject { |field| excluded_fields.include?(field.name) }
     end
 
-    def transformers(view_name)
-      views[view_name].transformers
+    def transformers_for(view_name)
+      view_transformers = Set.new(transformers(view_name))
```

Comment:
We haven't used `Set` in the project for anything else. Does it make sense just for this?

Reply:
Removed